example/src/main/scala/example/ContributorsCache.scala

```diff
@@ -17,45 +17,47 @@
 package example
 
 import example.ApiError._
-import sttp.client3.asynchttpclient.zio.{SttpClient, send}
+import sttp.capabilities.WebSockets
+import sttp.capabilities.zio.ZioStreams
 import sttp.client3.ziojson.asJson
-import sttp.client3.{UriContext, basicRequest}
+import sttp.client3.{SttpBackend, UriContext, basicRequest}
 import sttp.model.Uri
 import zio._
-import zio.duration._
 import zio.json._
-import zio.redis._
+import zio.redis.{Redis, _}
```

Comment:
Why was this necessary?

Reply:
I will delete `zio._` because it conflicts with the one in zio.